examples/draw_state.rs

```diff
@@ -0,0 +1,103 @@
+extern crate glutin_window;
+extern crate piston;
+extern crate graphics;
+extern crate gfx_graphics;
+extern crate find_folder;
+extern crate gfx;
+extern crate gfx_device_gl;
+
+use gfx::traits::*;
+use glutin_window::{GlutinWindow, OpenGL};
+use piston::window::{OpenGLWindow, Window, WindowSettings};
+use piston::event_loop::Events;
+use graphics::draw_state::Blend;
+use graphics::*;
+use piston::input::*;
+use gfx_graphics::{Flip, Gfx2d, Texture, TextureSettings};
+
+fn main() {
+    println!("Press A to change blending");
+    println!("Press S to change clip inside/out");
+
+    let opengl = OpenGL::V3_2;
+    let samples = 4;
+    let mut window: GlutinWindow = WindowSettings::new(
+            "piston: draw_state",
+            [600, 600]
+        )
+        .exit_on_esc(true)
+        .samples(samples)
+        .opengl(opengl)
+        .build()
+        .unwrap();
+
+    let (mut device, mut factory) = gfx_device_gl::create(|s|
+        window.get_proc_address(s) as *const std::os::raw::c_void);
```

Comment:
Notice that we no longer hold the window or wrap it into anything. This may need a revise on piston_window crates and such, potentially. Not to mention less headache for the users ;)